Update for new workflow

xoqhdgh1002 opened the pull request.

jpata commented:
Thanks!

Did you intend to delete [web/index.html](https://github.com/jpata/cms-reco-profiling-web/pull/66/files#diff-8f62b6ced28d3396b501d2e89a2e7cb761d16cd7dc977aebece03d4a5da5c24e)? Probably not, as that will delete the web page.
 
Please also remove the .pyc as before.

xoqhdgh1002 commented:
Thanks for comments!

For solve the problem for conflicted file, we deleted index.html and generated new html file.
Also, we removed the .pyc as before.

Please check and comment!

jpata commented:
Ah, looks like the github UI was somehow showing me only the removal of the file, but not the addition in https://github.com/jpata/cms-reco-profiling-web/pull/66/commits/ecfc2c7f5b53bef4794137232a7f76469140db7e.
